### PR TITLE
Be-13-add-redis-workflow

### PR DESCRIPTION
**Pull request by Randayy**
image
image

### CODE REVIEW

File: app/db/connect_redis.py [hunk 1]
```diff
@@ -20,6 +22,16 @@ async def get_redis(self):
...
+    
+    async def get_data(self, key):
+        redis = await self.get_redis()
+        data = await redis.get(key)
+        return data
+    
+    async def set_data(self, key, value,expire_time=172800):
+        redis = await self.get_redis()
+        await redis.set(key, value)
+        await redis.expire(key, expire_time)
```
Comment (Artemooon) [inline app/db/connect_redis.py]: you can use just 3 argument in the set method for the expire value
redis.set(key, value, expire)

Comment (Randayy) [reply app/db/connect_redis.py]: will fix this with all comments

File: app/db/connect_redis.py [hunk 2]
```diff
@@ -54,8 +66,8 @@ async def retrieve_data_from_redis():
...
             print("Not found")
 
-asyncio.run(store_data_in_redis())
+# asyncio.run(store_data_in_redis())
 
-asyncio.run(retrieve_data_from_redis())
+# asyncio.run(retrieve_data_from_redis())
 
-asyncio.run(check_redis_connection())
+# asyncio.run(check_redis_connection())
```
Comment (Artemooon) [inline app/db/connect_redis.py]: remove commented code

File: app/db/connect_redis.py (hunk 1 above)
Comment (Randayy) [reply app/db/connect_redis.py]: fixed

**Review by Artemooon — APPROVED** [review]

**Pull request merged**